gst/nnstreamer/tensor_converter/tensor_converter.c

```diff
@@ -1247,7 +1256,49 @@ gst_tensor_converter_parse_octet (GstTensorConverter * self,
     config->rate_d = 1;
   }
 
-  return (config->info.type != _NNS_END);
+  return (config->info.info[0].type != _NNS_END);
+}
+
+/**
+ * @brief Get sink pad template caps
+ */
+static GstCaps *
+gst_tensor_converter_get_template_caps (GstTensorConverter * self)
+{
+  GstCaps *caps = gst_caps_new_empty ();
+  const NNStreamerExternalConverter *ex;
+  gchar *plugin_name;
+  guint total, i;
+  subplugin_info_s info;
+  const gchar *prefix_str;
+  gsize prefix_len, extension_len, len;
+
+  /* append default caps */
+  append_video_caps_template (caps);
+  append_audio_caps_template (caps);
+  append_text_caps_template (caps);
+  append_octet_caps_template (caps);
+
+  /* get template caps from sub-plugins */
+  total = nnsconf_get_subplugin_info (NNSCONF_PATH_CONVERTERS, &info);
```

Comment:
These operations are not supposed to be required by plugins. You are doing something duplicated by changing the type to NNS_SEARCH_FILENAME